fenix/app/metrics.yaml

```diff
@@ -10399,3 +10399,313 @@ sync:
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: 122
+  bottom_sheet_learn_more_clicked:
+    type: event
+    description: |
+      The user clicked on learn more link from the onboarding card.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1843508
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3619
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: 122
+  bottom_sheet_expand_settings:
+    type: event
+    description: |
+      Settings card from the bottom sheet was expanded.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1843508
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3619
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: 122
+  bottom_sheet_no_review_reliability_available:
+    type: event
+    description: |
+      No analysis card was displayed to the user.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1843508
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3619
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: 122
+  bottom_sheet_analyze_reviews_none_available_clicked:
+    type: event
+    description: |
+      User clicked on the launch analyzer link from the no analysis card.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1843508
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3619
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: 122
+  bottom_sheet_reanalyze_clicked:
+    type: event
+    description: |
+      User interacted with the launch analyzer link from the "New info to check" warning card.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1843508
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3619
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: 122
+  bottom_sheet_reactivated_button_clicked:
+    type: event
+    description: |
+      User interacted with the button to report a product is back in stock.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1843508
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3619
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: 122
+
+shopping.settings:
+  settings_component_opted_out:
```

Comment:
Here you will want to remove the `settings_` prefix from the metrics' names.

In fact, does this successfully build? I would expect this to fail without a `no_lint`